carbonserver/trie.go

```diff
@@ -467,6 +468,19 @@ func (tn *trieNode) setChild(i int, n *trieNode) {
 	)
 }
 
+func (tn *trieNode) incrementFindMetric() {
+	if tn.file() {
+		meta := tn.meta.(*fileMeta)
+		atomic.AddInt64(&meta.readHits, 1)
+	}
+}
+func (tn *trieNode) incrementFindBytesMetric(bytesNumber int64) {
```

Comment:
nit-picking: `incrementRenderBytesMetric` might be better as only `/render/` api call return metric data.